src/Shared/EmptyListWrapper.js

```diff
@@ -0,0 +1,169 @@
+import React, { Component } from 'react'
+import { View, Text, StyleSheet, Image } from 'react-native'
+import Icon from 'react-native-vector-icons/dist/MaterialIcons'
+
+import WrappedTextButton from '../TextButton/TextButton.js'
+
+export default class EmptyListWrapper extends Component {
+  render() {
+    let { listEmptyState, items, children } = this.props
+
+    if (!listEmptyState) {
+      return <View>Loading...</View>
```

Comment:
You actually don't want this. For backward's compatibility's sake sometimes `listEmptyState` will always be undefined, and you don't want it to always say "Loading..."

Reply:
```suggestion
      return <>{children}</>
```